### DISCUSSION TITLE
Modal and Confirm Dialog HeaderCssClass/Dialog Doesnt Work

### DISCUSSION THREAD

**Discussion by mightygriffon6** [vikramlearning/blazorbootstrap#802] [Q&A] [answered]
This is my code and it should change the color but it doesnt, please help im using v. 3.0.0. preview.2
`<Modal @ref="modal" UseStaticBackdrop="true" CloseOnEscape="false" HeaderCssClass="custom-modal-header">

    <HeaderTemplate>
        @TituloModal
    </HeaderTemplate>
    `
   heres the css
`custom-modal-header {
    background-color: rgb(0,108,125);
    color: white;
}`
please give me an example of one well implemented or if its just a bug

**Comment by gvreddy04** [comment, marked as answer]
@mightygriffon6 Thank you for trying BlazorBootstrap. Here's an example below:

![image](https://github.com/user-attachments/assets/4f34e74e-4e4d-4c87-9d71-6dd170237888)

```cshtml
<Modal @ref="modal" Title="Modal title" HeaderCssClass="custom-modal-header">
    <BodyTemplate>
        Modal body text goes here.
    </BodyTemplate>
    <FooterTemplate>
        <Button Color="ButtonColor.Secondary" @onclick="OnHideModalClick">Close</Button>
        <Button Color="ButtonColor.Primary">Save changes</Button>
    </FooterTemplate>
</Modal>

<Button Color="ButtonColor.Primary" @onclick="OnShowModalClick">Show Modal</Button>

@code {
    private Modal modal = default!;

    private async Task OnShowModalClick()
    {
        await modal.ShowAsync();
    }

    private async Task OnHideModalClick()
    {
        await modal.HideAsync();
    }
}

<style>
    .custom-modal-header {
        background-color: rgb(0,108,125) !important;
        color: white !important;
    }
</style>
```
